swap/src/bin/swap.rs

```diff
@@ -77,6 +77,8 @@ async fn main() -> Result<()> {
             let bitcoin_wallet = Arc::new(bitcoin_wallet);
 
             let mut swarm = swarm::bob(&seed, alice_peer_id)?;
+            let our_peer_id = swarm.local_peer_id();
+            tracing::debug!("Our peer-id: {}", our_peer_id);
```

Comment:
Shall we move this to the `swarm::bob` function? Then we wouldn't have to duplicate it on the outside here.